### ISSUE TITLE
ModuleNotFoundError after 1.25 -> 1.29 update

### ISSUE THREAD

**Issue by slvrtrn** [trueaccord/scalapb-pants#14]
Hello! 
After updating Pants build system from 1.25 to 1.29 we started to see this error

```
Traceback (most recent call last):
  File "/home/user/.cache/pants/setup/bootstrap-Linux-x86_64/1.28.0_py36/lib/python3.6/site-packages/pants/init/extension_loader.py", line 279, in load_backend
    module = importlib.import_module(backend_module)
  File "/home/user/.cache/pants/setup/bootstrap-Linux-x86_64/1.28.0_py36/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/user/.cache/pants/plugins/scalapb.pants-0.4.0-py3-none-any.whl-install/scalapb/pants/register.py", line 3, in <module>
    from scalapb.pants.tasks.scalapb_gen import ScalaPBGen
  File "/home/user/.cache/pants/plugins/scalapb.pants-0.4.0-py3-none-any.whl-install/scalapb/pants/tasks/scalapb_gen.py", line 9, in <module>
    from twitter.common.collections import OrderedSet
ModuleNotFoundError: No module named 'twitter.common.collections'
18:14:20:453 [ERROR] Failed to load the scalapb.pants.register backend: ModuleNotFoundError("No module named 'twitter.common.collections'",)
(Use --print-exception-stacktrace to see more error details.)
```
looks like that dependency is now missing from Pants :(

**Comment by thesamet** [comment]
@slvrtrn Thanks for reporting. Will you be able to look into fixing this?

**Comment by slvrtrn** [comment]
> @slvrtrn Thanks for reporting. Will you be able to look into fixing this?

Will it be a good idea to just add this dependency to this project, as I am unsure about OrderedSet usage?

**Comment by slvrtrn** [comment]
@thesamet I've created a PR addressing this issue - https://github.com/scalapb/scalapb-pants/pull/15, could you please check it out?

**Comment by thesamet** [comment]
Thank you so much for looking into this!
1. Can you fix the error preventing the tests to run on Travis?
2. Regarding the warnings - the plugin is based on the protobuf plugin that comes with pants, perhaps making changes based on the file there would fix the warnings and prevent another hassle in the near future?

**Comment by thesamet** [comment]
Thanks for looking into this. Released this as 0.5.0
